zubhub_frontend/zubhub/src/assets/js/styles/views/profile/profileStyles.js

```diff
@@ -54,24 +70,50 @@ const styles = theme => ({
   tagsContainerStyle: {
     display: 'flex',
     flexWrap: 'wrap',
+    paddingTop: '0.5em',
+    paddingBottom: '0.5em',
+    [theme.breakpoints.down('657')]: {
+      paddingTop: '0',
+      paddingBottom: '0',
+    },
+  },
+  removeTagMargin: {
+    marginLeft: '0 !important',
+  },
+  centerTag: {
+  alignItems: 'center',
+  },
+  emailStyle: { 
+    overflowWrap: 'anywhere',
   },
-  emailStyle: { marginBottom: '0.5em' },
   dividerStyle: {
-    width: '100vw',
   },
   moreInfoBoxStyle: {
-    height: '3em',
     display: 'flex',
     flexDirection: 'row',
-    justifyContent: 'center',
-    alignItems: 'center',
+    justifyContent: 'space-evenly',
+    alignItems: 'end',
+    flexWrap: 'wrap',
+    [theme.breakpoints.down('657')]: {
+      marginTop: '1.3em',
+    },
   },
   moreInfoStyle: {
-    marginLeft: '0.5em',
-    marginRight: '0.5em',
-    fontWeight: 'bold',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+  },
+  moreInfoStyle1: {
+    fontWeight: '500',
     fontSize: '0.9rem',
     color: '#00B8C4',
+    paddingLeft: '5px',
+    paddingRight: '5px',
+  },
+  moreInfoStyle2: {
```

Comment:
This could be `moreInfoCount`?

Reply:
noted.

Reply:
> If there is no way to reconcile the distance, we might try to put them in some distinct containers to show some sort of separation? We can experiment with stuff like this as well:

Also, @tuxology I needed a little help in here, you see there is some padding involve on the Count_values which makes it look a little bit distant from the title. So I was trying to remove the padding but couldn't do much(padding is also not inherited, I tried inspecting it ). Can you suggest something or anything which I might be missing so that it can look more closer and similar to the mockup.

![Screenshot (42)](https://user-images.githubusercontent.com/92849065/176390658-af41c77b-ed42-4826-a7c8-1ea5c96f41a2.png)


![Screenshot (41)](https://user-images.githubusercontent.com/92849065/176387953-e704e8bf-dc4d-47ec-b5ef-4e86f5082a85.png)

Reply:
Maybe its not padding but `line-height`? maybe for this specific container, can you try to reduce it and see how it behaves?

Reply:
@Deepanshu039 OK, lets just proceed with this for now. We can do the design stuff later. I feel there is a lot of more stuff to implement for now such as Badges API, latest user activity (probably via API) etc. These are more pressing issues